### PR TITLE
Change panel size to absolute value, changed panel bullets to buttons, add banner to disabled buttons, cleanup img folders and scraps file

### PR DESCRIPTION
**Pull request by connorbo97**
- Panel size is now controlled by a constant
- Bullets are now buttons with icons
- Buttons that aren't impelemented yet have banners to imply they are disabled versus a different styling
- Removed some extra images in img folder and random scrap file that shouldn't have been committed

### CODE REVIEW

File: src/styles/Panel.css [hunk 1]
```diff
@@ -1,82 +1,148 @@
...
+  justify-content: flex-start;
+}
+
+.panel-options-list {
+  display: inline-block;
+  padding: 0px;
+  margin: 35px 25px;
+}
+
+/* .panel-options-item {
```
Comment (malsf21) [inline src/styles/Panel.css]: Is there a reason that these lines are commented out? Will we use this styling in the future?

File: src/styles/Panel.css [hunk 2]
```diff
@@ -1,82 +1,148 @@
...
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  color: white;
+  position: relative;
+  margin-bottom: 20px;
+}
+.panel-item-content {
+  position: relative;
+  font-size: 33px;
```
Comment (malsf21) [inline src/styles/Panel.css]: should we hard code w/ pixels or use em for font sizes, thoughts @connorbo97

Comment (connorbo97) [reply src/styles/Panel.css]: Still unsure of the px vs em relationship and when to use em tbh. Can you explain the main reason to switch?

File: src/styles/Panel.css (hunk 1 above)
Comment (connorbo97) [reply src/styles/Panel.css]: Those are the stylings for the bullet. It was mroe just in case the buttons didn't end up working out. I can remove em

**Pull request merged**